### PR TITLE
Create dashboards and squid

### PR DESCRIPTION
**Pull request by christopherjturner**

### CODE REVIEW

**Comment by christopherjturner** [comment]
image

File: src/server/services/transformers/service-status.js [hunk 1]
```diff
@@ -187,9 +191,101 @@ function serviceStatus(service) {
...
+      },
+      info: () => {
+        switch (cdpSquidProxy?.status) {
+          case creationStatuses.raised:
+          case creationStatuses.requested:
+          case creationStatuses.inProgress:
+            return `Setting up proxy access.`
+          case creationStatuses.created:
+          case creationStatuses.success:
+            return `Your test suite can make outbound calls via the proxy.`
```
Comment (feedmypixel) [inline src/server/services/transformers/service-status.js:215]: Should be `new service` not `test suite`

File: src/server/services/transformers/service-status.js [hunk 2]
```diff
@@ -187,9 +191,101 @@ function serviceStatus(service) {
...
+          ? startCase(cdpSquidProxy?.status)
+          : unknownValue,
+        classes: statusTagClassMap(cdpSquidProxy?.status)
+      },
+      info: () => {
+        switch (cdpSquidProxy?.status) {
+          case creationStatuses.raised:
+          case creationStatuses.requested:
+          case creationStatuses.inProgress:
+            return `Setting up proxy access.`
```
Comment (feedmypixel) [inline src/server/services/transformers/service-status.js:212]: doesn't need to be a template literal, no variable so single quotes are fine

File: src/server/services/transformers/service-status.js [hunk 3]
```diff
@@ -187,9 +191,101 @@ function serviceStatus(service) {
...
+          ? startCase(cdpDashboard?.status)
+          : unknownValue,
+        classes: statusTagClassMap(cdpDashboard?.status)
+      },
+      info: () => {
+        switch (cdpDashboard?.status) {
+          case creationStatuses.raised:
+          case creationStatuses.requested:
+          case creationStatuses.inProgress:
+            return `Setting up dashboards.`
```
Comment (feedmypixel) [inline src/server/services/transformers/service-status.js:258]: doesn't need to be a template literal, no variable so single quotes are fine

File: src/server/services/transformers/service-status.js [hunk 4]
```diff
@@ -187,9 +191,101 @@ function serviceStatus(service) {
...
+          case creationStatuses.created:
+          case creationStatuses.success:
+            return `Default dashboards created.`
```
Comment (feedmypixel) [inline src/server/services/transformers/service-status.js:261]: doesn't need to be a template literal, no variable so single quotes are fine

File: src/server/test-suites/transformers/test-suite-status.js [hunk 5]
```diff
@@ -74,9 +79,55 @@ function testSuiteStatus(service) {
...
+      },
+      info: () => {
+        switch (cdpSquidProxy?.status) {
+          case creationStatuses.raised:
+          case creationStatuses.requested:
+          case creationStatuses.inProgress:
+            return `Setting up proxy access.`
+          case creationStatuses.created:
+          case creationStatuses.success:
+            return `Your test suite can make outbound calls via the proxy.`
```
Comment (feedmypixel) [inline src/server/test-suites/transformers/test-suite-status.js:103]: doesn't need to be a template literal, no variable so single quotes are fine

**Review by feedmypixel — APPROVED** [review]
LGTM 👍 

- 1 typo
- a few pointers on when to use Template literals and when not to

File: src/server/services/transformers/service-status.js (hunk 4 above)
Comment (feedmypixel) [reply src/server/services/transformers/service-status.js:261]: Also can I suggest:

`Default Grafana observability metrics data and graph dashboards created for your service.`

File: src/server/test-suites/transformers/test-suite-status.js (hunk 5 above)
Comment (feedmypixel) [reply src/server/test-suites/transformers/test-suite-status.js:103]: Also can I suggest:

`Your new service can now make outbound calls via the proxy.`

File: src/server/services/transformers/service-status.js (hunk 3 above)
Comment (christopherjturner) [reply src/server/services/transformers/service-status.js]: currently everything in there is a template literal, might be worth fixing them all up in a clean-up pass at some point. There's a bit of duplication too, feels like we could make this block a bit smaller/cleaner.

**Pull request merged**